src/test/a11y/a11y.ts

```diff
@@ -22,7 +22,20 @@ class PallyIssue {
   typeCode: number;
 }
 
+function loginPally(): Pa11yResult {
+  return pa11y(config.TEST_URL + '/login', {
+    hideElements: '.govuk-footer__licence-logo, .govuk-header__logotype-crown',
+    actions: [
+      'set field #username to idamOwner@hmcts.net',
+      'set field #password to Pa55word11',
```

Comment:
When using the Idam on AAT to sign in the password is different. It is not good to hardcode the username and password on the test because our repo is opened  to public on github. Yourshould be getting them from Azure keyvault. We need to find out if those values are in the keyvault or not first and pull them in using Helm chart and 'property volume' if they do.